Week1/index.js

```diff
@@ -0,0 +1,209 @@
+                             Netherlands?"
+7          shows answer of "What's the population of Rotterdam?"
+8          shows answer of "What's the top 10 countries based on surface area?"
+9          shows answer of "What's the top 10 cities with the highest population?"
+10         shows answer of "What's the population of the world ?"
+  `);
+};
+
+const queries = [
+  {
+    query: `select name,population from countries where population > 8000000;`,
+    question: 1,
+  },
+  {
+    query: `select name from countries where name like '%land%';`,
+    question: 2,
+  },
+  {
+    query: `select name,population from cities where population >500000 and population < 1000000;`,
+    question: 3,
+  },
+  {
+    query: `select name from countries where continent = 'Europe';`,
+    question: 4,
+  },
+  {
+    query: `select * from countries order by surface_area desc;`,
+    question: 5,
+  },
+  {
+    query: `select name,population from cities where country_code='NLD';`,
+    question: 6,
+  },
+  {
+    query: `select name,population from cities where name='Rotterdam';`,
+    question: 7,
+  },
+  {
+    query: `select name,surface_area from countries order by surface_area desc limit 10;`,
+    question: 8,
+  },
+  {
+    query: `select name,population from cities order by population desc limit 10;`,
+    question: 9,
+  },
+  {
+    query: `select sum(population) as world_population from countries;`,
+    question: 10,
+  },
+];
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'rootuser',
+});
+
+const execQuery = promisify(connection.query.bind(connection));
+
+const consoleQuery = async question => {
+  queries.filter(query => {
+    if (query.question === question) {
+      console.log(`ANSWER OF QUESTION ${query.question} :`);
+      return execQuery(query.query).then(console.table);
+    }
+  });
+};
+
+async function seedDatabase() {
+  const DELETE_DATABASE = `DROP DATABASE IF EXISTS WORLD;`;
+  const CREATE_DATABASE_WORLD = `CREATE DATABASE IF NOT EXISTS world;`;
+  const USE_DATABASE_WORLD = `use world;`;
+  const CREATE_CITIES_TABLE = `
+  CREATE TABLE IF NOT EXISTS cities (
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(254),
+    country_code VARCHAR(3),
+    district VARCHAR(254),
+    population INT,
+    PRIMARY KEY (id)
+    );`;
+  const CREATE_COUNTRIES_TABLE = `
+    CREATE TABLE IF NOT EXISTS countries (
+      id INT NOT NULL AUTO_INCREMENT,
+      name VARCHAR(254),
+      continent enum('Asia', 'Africa', 'North America', 'South America', 'Oceania', 'Europe', 'Antarctica'),
+      region VARCHAR(254),
+      surface_area FLOAT,
+      indep_year INT,
+      population INT,
+      life_expectancy FLOAT,
+      gnp FLOAT,
+      gnp_old FLOAT,
+      local_name VARCHAR(254),
+      government_form VARCHAR(254),
+      head_of_state VARCHAR(254),
+      capital INT,
+      PRIMARY KEY (id)
+      );`;
+
+  connection.connect();
```

Comment:
It's always a good idea to add validation on everything that could go on. And connecting to the DB is one of those places where tons of things can go wrong: user does not exist, wrong password, database does not exist, etc

Reply:
hey @roxana-lafuente . Thank you for your helpful review. you're right, I also have to do that but I tried to upload homework as soon as possible. I will work on it.